src/main/java/org/kohsuke/github/GHCheckRun.java

```diff
@@ -29,7 +30,7 @@
     private URL detailsUrl;
     private Output output;
     private GHApp app;
-    private GHPullRequest[] pullRequests;
+    private List<GHPullRequest> pullRequests;
```

Comment:
I think I'd rather keep the array here and have `getPullRequests()` return an `UnmodifiableList` created from it.  
Or maybe return an `Iterable`.  

I don't think there's a strong reason to change to a List, but I'm open to discussing it.

Reply:
I do not have any strong opinions on the need to return an `UnmodifiableList` / `Iterable`  vs changing this to a `List`. Will update 😄